<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>720</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="settingsWidget" native="true">
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-20</y>
         <width>858</width>
         <height>571</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget_1" native="true">
          <widget class="QComboBox" name="movingBlockColorComboBox">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>50</y>
             <width>180</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>红色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>橙色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>黄色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>绿色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>蓝色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>粉色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>紫色</string>
            </property>
           </item>
          </widget>
          <widget class="QComboBox" name="fixedBlockColorComboBox">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>125</y>
             <width>180</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>红色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>橙色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>黄色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>绿色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>蓝色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>粉色</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>紫色</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="fixedBlockColorLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>125</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>固定方块颜色：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QSpinBox" name="widthSpinBox">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>275</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="heightSpinBox">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>350</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
          <widget class="QLabel" name="movingBlockColorLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>50</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>移动方块颜色：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="widthLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>275</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>棋盘宽度：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="heightLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>350</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>棋盘高度：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="blockSpeedLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>200</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>方块下落速度：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="musicChoiceLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>425</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>背景音乐曲目：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="musicVolumeLabel">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>500</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>背景音乐音量：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QSlider" name="musicVolumeSlider">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>500</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QComboBox" name="musicChoiceComboBox">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>425</y>
             <width>180</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>上古卷轴</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>植物大战僵尸</string>
            </property>
           </item>
          </widget>
          <widget class="QSlider" name="blockSpeedSlider">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>200</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <widget class="QKeySequenceEdit" name="rorateKeyEdit">
           <property name="geometry">
            <rect>
             <x>225</x>
             <y>200</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
          <widget class="QKeySequenceEdit" name="moveLeftKeyEdit">
           <property name="geometry">
            <rect>
             <x>225</x>
             <y>50</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
          <widget class="QKeySequenceEdit" name="moveRightKeyEdit">
           <property name="geometry">
            <rect>
             <x>225</x>
             <y>125</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
          <widget class="QLabel" name="moveLeftLabel">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>50</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>左移按键：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="moveRightLabel">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>125</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>右移按键：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="rorateLabel">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>200</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>旋转按键：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="pauseLabel">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>350</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>暂停按键：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="quickenLabel">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>275</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>加速按键：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="pictureChoiceLabel">
           <property name="geometry">
            <rect>
             <x>75</x>
             <y>425</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>背景图片：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QComboBox" name="pictureChoiceComboBox">
           <property name="geometry">
            <rect>
             <x>225</x>
             <y>425</y>
             <width>180</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>纯白</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>印象 - 日出</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>印象 - 日落</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>印象 - 森林</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>印象 - 沙漠</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>印象 - 星空</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>上古卷轴5：天际</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>植物大战僵尸</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>苦力怕</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>你的名字。</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>泛舟</string>
            </property>
           </item>
          </widget>
          <widget class="QKeySequenceEdit" name="pauseKeyEdit">
           <property name="geometry">
            <rect>
             <x>225</x>
             <y>350</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
          <widget class="QKeySequenceEdit" name="quickenKeyEdit">
           <property name="geometry">
            <rect>
             <x>225</x>
             <y>275</y>
             <width>150</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="saveButton">
       <property name="geometry">
        <rect>
         <x>375</x>
         <y>575</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>保存设置</string>
       </property>
      </widget>
      <widget class="QPushButton" name="returnToMenuButton">
       <property name="geometry">
        <rect>
         <x>625</x>
         <y>575</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回主菜单</string>
       </property>
      </widget>
      <widget class="QPushButton" name="resetButton">
       <property name="geometry">
        <rect>
         <x>125</x>
         <y>575</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>重置设置</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
